app/lib/shared/analyzer_memcache.dart

```diff
@@ -40,10 +47,23 @@ class AnalyzerMemcache {
     return _data.setText(_dataKey(package, version, panaVersion), content);
   }
 
+  Future<String> getExtract(String package, String version) {
+    return _data.getText(_extractKey(package, version));
```

Comment:
Shouldn't this be `_extract.getText()`? (ditto below)

Reply:
good catch, thanks!